docs/chain/collator.md

```diff
@@ -210,53 +192,50 @@ docker run -p 127.0.0.1:9933:9933 -v ~/data:/data \
     --execution=wasm
 ```
 
-The docker commands will map the database files for the relay and parachain as well as the keystore directory to `~/data` on your host system using the flag `-v $HOME/data:/data`.
-That way you can make sure to not lose those files when you remove the container.
+If the node needs to be reachable via PolkadotJS Apps, the `--ws-external` flag must be added to the collator options, before the `--` divider, and the WS port must be exposed from the container with an additional `-p 9944:9944` parameter.
+
+The docker command will map the database files for the relay and parachain as well as the keystore directory to `~/data` on the host system using the flag `-v $HOME/data:/data`.
+That way the blockchain database files are not lost when and if the Docker container is removed and can be mounted back on next containers.
 
-The docker container runs as an user with id 1000.
-The container will try to access the mapped volume and the files in it.
+The docker container runs as an user with id 1000 and will try to access the mapped volume and the files it contains.
 If the files are not owned by a user with id 1000 this will result in an error.
-Run `chown -R 1000:1000 $HOME/data` to give the container access.
+If that is the case, run `sudo chown -R 1000:1000 $HOME/data` to give the container access.
 
 
 </TabItem>
 </Tabs>
 
-## Sync Data
-
-Before a Collator can author blocks, the node needs to fully sync up with the blockchain.
-Depending on which chain and the size of it may take a number of minutes to several hours maybe even a day.
+## Sync the Blockchain State
 
-More details can be found at the [Polkadot network](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-kusama#synchronize-chain-data)
+Before a collator can author blocks, the node needs to fully sync up with both the parachain and the relaychain.
+Depending on the size of the blockchain states, it may take a number of minutes to several hours maybe even days for the node to catch up.
+More details can be found on the [Polkadot network docs](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-kusama#synchronize-chain-data).
```

Comment:
Don't give them false hopes.

```suggestion
Depending on the size of the blockchain states, it may take a number of hours to a couple of days for the node to catch up.
More details can be found on the [Polkadot network docs](https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-kusama#synchronize-chain-data).
```

Reply:
I did not change that bit, I just copied it over from further down the text 😂

Reply:
would be cool to have a switch in the beginning of the doc and then we just show either peregrine or spiritnet data where appropriate. Like a Tab over the whole document. :D